struct.c

```diff
@@ -316,25 +316,16 @@ rb_struct_s_inspect(VALUE klass)
     return inspect;
 }
 
-static VALUE
-struct_new_kw(int argc, const VALUE *argv, VALUE klass)
-{
-    return rb_class_new_instance_kw(argc, argv, klass, RB_PASS_CALLED_KEYWORDS);
-}
-
 static VALUE
 setup_struct(VALUE nstr, VALUE members, int keyword_init)
```

Comment:
Maybe `keyword_init` argument is no longer used?

Reply:
`keyword_init` still has an effect in `Struct#initialize` (aka `rb_struct_initialize_m`), but shouldn't have an impact on `new` (which should simply allocate the object an call `initialize`).

Or did I misunderstand your comment?

Reply:
Oh, I see. You are right, we no longer need the argument.

Reply:
Commit amended 👍